src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/ContextInfos/PackageSearchMetadataContextInfo.cs

```diff
@@ -33,7 +35,7 @@ public sealed class PackageSearchMetadataContextInfo
         public long? DownloadCount { get; internal set; }
         public IReadOnlyCollection<PackageDependencyGroup>? DependencySets { get; internal set; }
         public LicenseMetadata? LicenseMetadata { get; internal set; }
-        public PackageReaderBase? PackageReader { get; internal set; }
+        public string? PackagePath { get; internal set; }
```

Comment:
remove `?` since strings are already nullable

Reply:
After some reading, apparently it's a trend now to use ? on a string to indicate you expect null sometimes. I think that's a bit cryptic, but were you going for that?

Reply:
@donnie-msft the NuGet.VisualStudio.Internal.Contracts project uses [C#'s nullable reference types feature](https://docs.microsoft.com/en-us/dotnet/csharp/nullable-references).

Reply:
We should add coding guidance and a plan for how we're refactoring old code before C# 8.0